packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -332,48 +321,38 @@ export class ProfilesUtils {
                 await this.promptAndHandleMissingCredentialManager(credentialManagerMap);
             }
             if (credentialManagerMap && isVSCodeCredentialPluginInstalled) {
-                return this.setupCustomCredentialManager(credentialManagerMap);
+                return new imperative.ProfileInfo("zowe", {
+                    credMgrOverride: await this.setupCustomCredentialManager(credentialManagerMap),
+                });
             }
-            return this.setupDefaultCredentialManager();
         }
 
-        const profileInfo = new imperative.ProfileInfo("zowe", {});
-        const workspacePath = ZoweVsCodeExtension.workspaceRoot?.uri.fsPath;
-        // Trigger initialize() function of credential manager to throw an error early if failed to load
-        await profileInfo.readProfilesFromDisk({
-            homeDir: FileManagement.getZoweDir(),
-            projectDir: workspacePath ? FileManagement.getFullPath(workspacePath) : undefined,
+        return new imperative.ProfileInfo("zowe", {
+            credMgrOverride: hasSecureCredentialManagerEnabled && (await this.setupDefaultCredentialManager()),
```

Comment:
I like the `&&` 😋 
but I'm curious if we should convert this to a full ternary operator and fallback to `undefined` instead of `false` 🤔 

Technically, it shouldn't cause troubles since this sort of thing is handled by the `imperative.ProfileCredentials`
`await CredentialManagerFactory.initialize({ service: null, ...this.mCredMgrOverride || {} });`

I'm curious how some static analysis tools will react to us possibly setting `credMgfOverride: false` 😋

Reply:
After some further discussion and testing, this was a problem with my `.env` file where I had `ZOWE_CLI_HOME` set to my current `.dist`. That meant that zowe events needed to happen there in order for ZE to capture them 😋 

Thanks @t1m0thyj

Reply:
thanks for addressing this 🙏